dev-libs/libthreadar/libthreadar-1.3.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Threading library used by dar archiver"
+HOMEPAGE="https://sourceforge.net/projects/libthreadar/"
+SRC_URI="mirror://sourceforge/libthreadar/${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux"
```

Comment:
We need to drop to just ~amd64 and request rekeywording of the new version of dar.